### DISCUSSION TITLE
WinUi3 Window Help

### DISCUSSION THREAD

**Discussion by jarno9981** [dotMorten/WinUIEx#131] [Q&A]
Using  WinUiEx 2.2.0

i have uwp project 

using this code

  var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
  coreTitleBar.ExtendViewIntoTitleBar = true;

  coreTitleBar.LayoutMetricsChanged += CoreTitleBar_LayoutMetricsChanged;

But When I Use it in WinUi3 With WinUiEx then it gives error .GetCurrentView().TitleBar; no element 
found using this Code For titlebar

 App.MainWindow.ExtendsContentIntoTitleBar = true;            
 App.MainWindow.SetTitleBar(CustomDragRegion);
 App.MainWindow.Activated += MainWindow_Activated;
 App.MainWindow.Title = "FireBrowser WinUi3";        

but i cant find LayoutMetricsChanged i want to use the method the use

``    if (FlowDirection == FlowDirection.LeftToRight)
    {
        CustomDragRegion.MinWidth = sender.SystemOverlayRightInset;
    }
    else
    {
        CustomDragRegion.MinWidth = sender.SystemOverlayLeftInset;

    }

    // Ensure that the height of the custom regions are the same as the titlebar.
    CustomDragRegion.Height =  sender.Height;`

without CoreTitleBar_LayoutMetricsChanged i cant use flowdirection

**Comment by dotMorten** [comment]
@jarno9981 I'm a bit confused about your question. I don't really see any WinUIEx APIs being used above.

**Reply by jarno9981** [reply]
![20230901_181952.jpg](https://github.com/dotMorten/WinUIEx/assets/53493418/ded7acf9-0406-4af7-9515-fea85642fe3c)

I want to get the current titlebar so i can use overflow but when using WinUiEx it wont work but normal Window will work

**Reply by dotMorten** [reply]
Thank you. That wasn't clear that the difference is WinUIEx Window vs WinUI3 Window. You were only comparing to your UWP version above.
Don't set the Content to null. That'll mess with WinUIEx's custom handling of titlebar.

Btw, I'd actually recommend that you _don't_ use `WindowEx`, but instead use the regular `Window` and use WinUIEx's `WindowManager` and extension methods to get at all the Window enhancements.

**Reply by jarno9981** [reply]
But WindowEx has better titlebar costimaztion and window handling then normal window that why i am using WindowEx

**Reply by jarno9981** [reply]
do i need to set the content to a page or anything

**Reply by dotMorten** [reply]
Set the content in your xaml
